src/XRP/default-xrp-client.ts

```diff
@@ -43,14 +38,14 @@ import { LedgerSpecifier } from './Generated/web/org/xrpl/rpc/v1/ledger_pb'
 import SendXrpDetails from './model/send-xrp-details'
 import { AccountSetFlag } from './model/account-set-flag'
 import TransactionResult from './model/transaction-result'
-
-/** A margin to pad the current ledger sequence with when submitting transactions. */
-const maxLedgerVersionOffset = 10
+import CommonXrplClient from './common-xrpl-client'
 
 /**
  * DefaultXrpClient is a client which interacts with the XRP Ledger.
  */
 export default class DefaultXrpClient implements XrpClientDecorator {
+  public commonXrplClient: CommonXrplClient
```

Comment:
can we make this private? seems bad if callers are using the commonXrplClient

Reply:
apparently we can!  the `DefaultXrpClient` is private by the time the public API is exposed but I agree that this is better practice.